src/core/public/application/types.ts

```diff
@@ -41,6 +48,11 @@ export interface AppBase {
    */
   title: string;
 
+  /**
+   * The category the app lives in
+   */
+  category: AppCategory;
+
```

Comment:
1. I should actually change this to optional. I switched it to required while fighting with typescript...

Reply:
2. Unless someone can walk me through it and it's a pretty trivial add, I'm thinking to punt on creating the API and limit the scope of this for 7.6. @alexfrancoeur is ok punting on this for 7.6 in favor of limiting scope. If someone can walk me through it and it's a pretty trivial add, I can try to sneak it in but otherwise I'm inclined to punt on it.

Reply:
First point can probably be done at a later time. As it's UI independent, it could even be handled by the platform team.

The second point was more a question of do we actually want to allow custom categories or not?

Reply:
> The second point was more a question of do we actually want to allow custom categories or not?

Yeah, eventually we do